docs/pages/color_usage.js

```diff
@@ -0,0 +1,270 @@
+import PageHeader from '../components/PageHeader.js';
+import ColorTile from '../components/ColorTile.js';
+import Page from '../components/Page.js';
+
+export default function ColorUsagePage(): Node {
+  return (
+    <Page title="Color Usage">
+      <PageHeader
+        name="Color usage"
+        description={`
+    Our color palettes are shared between Brand and Gestalt, and represent our full range of options. Please refer to the [Color Palette page](/color_palette) for all the shades and tints available.
+
+    On this page, you will find guidance on how to use our color tokens for:
+
+    - Primary and secondary elements
+    - Additional UI elements, including colors for shopping, messages, typography, and iconography
+    - Accessibility and inclusive design
+    `}
+        showSourceLink={false}
+      />
+      <MainSection
+        name="Applying color to UI"
+        description={`
+        When designing a product interface, you can choose between any color swatch available in our [design tokens documentation](/design_tokens). The color tokens represent the values used within Gestalt to construct layouts and components.
+
+        Colors available in our extended palette and not listed on Gestalt Colors & Styles Figma library are reserved for larger brand moments, and they should not be applied to components.
+
+        By limiting the range of colors, the areas that receive color gain more attention, such as text, images, and components. Pinterest content is multicolored, and having a limited range of colors is a great way to keep consistency, making our product more visible and engaging.
+
+        If there is a need for a color not available in our design tokens, please reach out to the Core Brand team for guidance on using color to propose a brand moment within the product UI.
+        `}
+      />
+      <MainSection
+        name="Primary color"
+        description={`
+        A primary color is the color displayed most frequently across screens and components, used to convey high-emphasis actions. Our primary color related tokens are:
+        **$color-background-primary-base** - Use this token when conveying a primary action.
+        **$color-background-brand** - Use this token when a background color is needed to signify the Pinterest brand.        `}
+      >
+        <ColorTile
+          description="Primary base"
+          textColor="white"
+          fullTokenName="color-background-primary-base"
+        />
+      </MainSection>
+      <MainSection
+        name="Secondary color"
+        description={`
+        A secondary color highlights medium to low-emphasis actions, creating a balance with the primary color. Our secondary color related token is:
+        **$color-background-secondary-base** - Use this token for secondary, medium to low-emphasis actions.
+        `}
+      >
+        <ColorTile description="Secondary base" fullTokenName="color-background-secondary-base" />
```

Comment:
This needs dark text — I can barely read it
![Screen Shot 2022-02-16 at 11 18 22 AM](https://user-images.githubusercontent.com/12059539/154339503-16ce3523-774f-41bd-8b4e-93ca73010f9b.png)